doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,230 @@ temperature:
+
+Cantera comes with a broad variety of well-commented example scrips for reactor networks. Please refer to them for further information (`Python <cython/examples.html>`_, `Matlab <matlab/examples.html>`_).
+
+
+Common Reactor Types and their Implementation in Cantera
+========================================================
+
+
+Batch Reactor at Constant Volume or at Constant Pressure
+--------------------------------------------------------
+
+If you are interested in how a homogeneous chemical composition changes in time when it is left to its own, a simple batch reactor can be used. Two versions are commonly considered: A rigid vessel with fixed volume but variable pressure, or a system idealized at constant pressure but varying volume.
+
+In Cantera, such a simulation can be performed very easily. The initial state of the solution can be specified by composition and a set of thermodynamic parameters (like temperature and pressure) as a standard Cantera solution object. Upon its base, a general (Ideal Gas) Reactor or an (Ideal Gas) Constant Pressure Reactor can be created, depending on if a constant volume or constant pressure batch reactor should be considered, respectively. The behavior of the solution in time can be simulated as a very simple Reactor Network containing only the formerly created reactor.
+
+An example for such a Batch Reactor is `reactor1.py <cython/examples/reactors_reactor1.html>`_.
+
+
+Continuously Stirred Tank Reactor
+---------------------------------
+
+In literature, Continuously Stirred Tank Reactors (CSTR) are often used to perform basic computations like blowout limits in combustion systems. They are also often referred to as Well-Stirred Reactor (WSR), or Perfectly Stirred Reactor (PSR), or Longwell Reactor.
+
+A CSTR assumes a confined volume in which the fluid is homogeneous at all points. A stream of reactants enter the confinement at a steady mass flow rate `\dot{m}`. The reactor is assumed to be steady. Therefore, the identical mass flow rate `\dot{m}` of products have to exit the confinement. The mass contained in the confinement `m` divided by the mass flow rate `\dot{m}` defines the mean residence time of the fluid in the confinement.
+
+The description is almost identical to the definition of a general Cantera reactor with a fixed volume. Therefore, the equations given in `Governing Equations for Single Reactors`_ apply, except that all unsteady terms are zero.
+
+Due to the assumption of a steady problem, the system of ordinary differential equations can be reduced to a set of coupled nonlinear algebraic equations. A Newton solver could be used to solve this system of equations. However, a sophisticated implementation might be required to account for stiffness.
```

Comment:
I agree - I've only ever seen "stiffness" used to refer to a large span in
*time* constants for different processes.  Since we are explicitly solving
algebraic equations, this would seem to be quite right.

On Tue, Jun 30, 2015 at 2:18 PM, Ray Speth <notifications@github.com> wrote:

> In doc/sphinx/reactors.rst
> <https://github.com/Cantera/cantera/pull/281#discussion_r33618593>:
>
> > +
> > +In Cantera, such a simulation can be performed very easily. The initial state of the solution can be specified by composition and a set of thermodynamic parameters (like temperature and pressure) as a standard Cantera solution object. Upon its base, a general (Ideal Gas) Reactor or an (Ideal Gas) Constant Pressure Reactor can be created, depending on if a constant volume or constant pressure batch reactor should be considered, respectively. The behavior of the solution in time can be simulated as a very simple Reactor Network containing only the formerly created reactor.
> > +
> > +An example for such a Batch Reactor is `reactor1.py <cython/examples/reactors_reactor1.html>`_.
> > +
> > +
> > +Continuously Stirred Tank Reactor
> > +---------------------------------
> > +
> > +In literature, Continuously Stirred Tank Reactors (CSTR) are often used to perform basic computations like blowout limits in combustion systems. They are also often referred to as Well-Stirred Reactor (WSR), or Perfectly Stirred Reactor (PSR), or Longwell Reactor.
> > +
> > +A CSTR assumes a confined volume in which the fluid is homogeneous at all points. A stream of reactants enter the confinement at a steady mass flow rate `\dot{m}`. The reactor is assumed to be steady. Therefore, the identical mass flow rate `\dot{m}` of products have to exit the confinement. The mass contained in the confinement `m` divided by the mass flow rate `\dot{m}` defines the mean residence time of the fluid in the confinement.
> > +
> > +The description is almost identical to the definition of a general Cantera reactor with a fixed volume. Therefore, the equations given in `Governing Equations for Single Reactors`_ apply, except that all unsteady terms are zero.
> > +
> > +Due to the assumption of a steady problem, the system of ordinary differential equations can be reduced to a set of coupled nonlinear algebraic equations. A Newton solver could be used to solve this system of equations. However, a sophisticated implementation might be required to account for stiffness.
>
> Is stiffness the correct term here? I think I've only ever seen this used
> to describe differential equations. The problems with the steady state
> solution are more about the strong nonlinearities and the presence of
> multiple solutions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cantera/cantera/pull/281/files#r33618593>.
>